app/src/main/res/layout/fragment_achievements.xml

```diff
@@ -6,542 +6,544 @@
+            android:layout_marginRight="@dimen/large_gap"
+            android:layout_marginEnd="@dimen/large_gap"
+            android:progress="50"
+            app:progress_text_size="@dimen/progressbar_text"
+            android:id="@+id/images_used_by_wiki_progress_bar"
+            app:progress_end_color="#8C8B98"
+            app:progress_start_color="#3A3381"
+            app:progress_stroke_width="2.5dp"
+            app:progress_text_color="@color/secondaryColor"
+            app:progress_text_format_pattern="12/24"
+            app:style="solid_line" />
+
+        </RelativeLayout>
+
+        <ProgressBar
+          android:id="@+id/progressBar"
+          android:layout_centerVertical="true"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content" />
+
+      </RelativeLayout>
+
+      <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:id="@+id/layout_statistics"
+        android:orientation="vertical">
+
+        <TextView
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:text="@string/statistics"
+          style="?android:textAppearanceLarge"
+          android:layout_marginLeft="@dimen/activity_margin_horizontal"
+          android:layout_marginStart="@dimen/activity_margin_horizontal"
+          android:layout_marginTop="@dimen/activity_margin_vertical"
+          android:textAllCaps="true"/>
+
+        <RelativeLayout
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:layout_marginLeft="@dimen/activity_margin_horizontal"
+          android:layout_marginRight="@dimen/activity_margin_horizontal"
+          android:layout_marginEnd="@dimen/activity_margin_horizontal"
+          android:layout_marginStart="@dimen/activity_margin_horizontal"
+          android:layout_marginTop="@dimen/activity_margin_horizontal">
+
+          <LinearLayout
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_centerVertical="true"
+            android:layout_alignParentStart="true"
+            android:layout_alignParentLeft="true"
+            android:layout_toStartOf="@+id/wikidata_edits"
+            android:layout_toLeftOf="@+id/wikidata_edits"
+            android:orientation="horizontal"
+            android:gravity="center_vertical">
+
+            <ImageView
+              android:layout_width="@dimen/overflow_icon_dimen"
+              android:layout_height="@dimen/overflow_icon_dimen"
+              android:id="@+id/wikidata_edits_icon"
+              app:srcCompat="@drawable/ic_custom_map_marker" />
 
             <TextView
-              style="?android:textAppearanceMedium"
-              android:layout_width="wrap_content"
+              android:layout_width="0dp"
+              android:layout_weight="0.8"
```

Comment:
hey @isdl03 , thanks for these changes. However this is one of few related lines with the issue. Why all these lines are edited? Can you please make sure just related edits are made?

Reply:
@neslihanturan We have made the requested changes. Kindy review them.